docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -36,19 +36,17 @@ To create a Grafana Mimir or Loki managed recording rule
 
 1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page listing existing alerts.
 1. Click **New alert rule**. The new alerting rule page opens where the **Grafana managed alert** option is selected by default.
-1. In Step 1, add the rule name.
-   - In **Rule name**, add a descriptive name. This name is displayed in the alert rule list. It is also the `alertname` label for every alert instance that is created from this rule.
+1. In Step 1, add the rule name. Recording rule name must be valid metric name.
```

Comment:
```suggestion
1. In Step 1, add the rule name. The recording rule name must be a valid metric name.
```

Reply:
What does "must be a valid metric name" mean? could we maybe write this more clearly?

Reply:
means that Recording rule names must also be valid prometheus metric names, that is, contain no whitespace. Maybe we could add this requirement 👍